jsapp/js/dataInterface.d.ts

```diff
@@ -7,6 +7,45 @@ interface FailResponse {
   statusText: string
 }
 
+interface SubmissionAttachment {
```

Comment:
I'm wondering what your process is for finding all the possible attributes for backend responses? Do you scan the code and see what we use or are you checking somewhere in the backend/api?

Reply:
I'm looking at the endpoint responses and build the `interface`s with that. Then after I write some more TypeScript it turns out some stuff is missing and I add it.

Reply:
Makes sense, that's what I had in mind as well :+1: